src/svgutils/transform.py

```diff
@@ -49,6 +49,45 @@ def rotate(self, angle, x=0, y=0):
         self.root.set("transform", "%s rotate(%f %f %f)" %
                       (self.root.get("transform") or '', angle, x, y))
 
+    def skew_x(self, angle):
```

Comment:
do we really need the `skew_x` and `skew_y` functions? Would it be enough to use `skew` with 0 in unwanted axis (like `skew(x, y=0)` or `skew(x=0, y)` ?